Foundation/NSHTTPCookieStorage.swift

```diff
@@ -76,8 +87,9 @@ open class HTTPCookieStorage: NSObject {
         if directory(with: path) {
             return path + fileName
         }
-        //if we were unable to create the desired directory, create the cookie file in the `pwd`
-        return FileManager.default.currentDirectoryPath + fileName
+        //if we were unable to create the desired directory, create the cookie file
+        //in a subFolder (named after the bundle) of the `pwd`
+        return FileManager.default.currentDirectoryPath + "/" + bundleName + fileName
```

Comment:
If it's a subfolder, shouldn't there be a `/` between the `bundleName` and `fileName`?